src/components/TopBar/TopBar.vue

```diff
@@ -462,18 +435,23 @@ export default {
 	}
 
 	.talk-sidebar-callview & {
-		margin-inline-end: var(--default-clickable-area);
+		padding-inline-end: calc(var(--default-clickable-area) + var(--default-grid-baseline) * 3);
```

Comment:
These styles are only working in Files Sidebar for logged-in users (`.talk-sidebar-callview`), but not for guest shares. Consider moving them all to `.top-bar--sidebar`